userspace/libscap/engine/bpf/scap_bpf.c

```diff
@@ -1849,22 +1856,20 @@ const struct metrics_v2* scap_bpf_get_stats_v2(struct scap_engine_handle engine,
 				{
 					strlcpy(stats[offset].name, info.name, METRIC_NAME_MAX);
 				}
+				strlcat(stats[offset].name, bpf_libbpf_stats_names[stat], sizeof(stats[offset].name));
```

Comment:
[nit] re comment here https://github.com/falcosecurity/libs/pull/2031/files#diff-12833abd4271488260dae0ba178c6ad3f0bc63642f793a20b06ab4eb10d02cf9L1839 libbpf stats were introduced w/ kernel 5.1 so folks with lower kernels can't reach this code since we check for libbpf stats being enabled.

Reply:
You are right but since usually many bpf features are backported I'm not so confident in removing it... I found this commit https://github.com/falcosecurity/libs/pull/1044/commits/957ab1c8577e6ef1505033717ec17015c8a18e36, unfortunately, I don't remember why I added it but i bet I had found an issue on some old machines...

Reply:
Fair, yes the backports.